### ISSUE TITLE
An environment variable for the standard modules?

### ISSUE THREAD

**Issue by konnov** [tlaplus/tlaplus#490]
> Run TLC with -DTLA-Library=/path/to/lib/archive or add the library archive to the Toolbox (File > Preferences > TLA+ Preferences > TLA+ library path locations).

It would be great to have a standard environment variable, e.g., called `TLA_PATH`. Similar to `PATH`, `PYTHON_PATH`, `JAVA_LIBRARY_PATH`, etc., it would list directories, from which the TLA+ tools would pull the library modules.

**Comment by lemmy** [comment]
Does @informalsystems have the resources to provide a pull request?

**Comment by konnov** [comment]
In principle, yes, but I am not sure about the whereabouts of the file reading operations in the SANY parser.

**Comment by lemmy** [comment]
util.FilenameToStream and its subclasses such as util.SimpleFilenameToStream

Since `TLA_PATH` is an alternative to "TLA-Library", it should work for SANY and the other tools such as TLC.

**Comment by konnov** [comment]
Thanks! I will have a look.

**Comment by lemmy** [comment]
@konnov Are you still pushing this?

**Comment by konnov** [comment]
Not really. I was stuck at how Eclipse handles filenames. So I guess the related PR #493 needs some expertise in Eclipse.

**Comment by ahelwer** [comment]
@lemmy what do you think about also adding this as a TLC CL option? You can define additional directories to search for TLA files when running TLC.
